src/routes/webhooks/paddle.ts

```diff
@@ -97,6 +163,23 @@ export const paddle = async (fastify: FastifyInstance): Promise<void> => {
                   data: eventData,
                   state: false,
                 });
+                await logPaddleAnalyticsEvent(
+                  eventData,
+                  AnalyticsEventName.CancelSubscription,
+                );
+                break;
+              case EventName.SubscriptionUpdated:
+                await logPaddleAnalyticsEvent(
```

Comment:
Is it always the case for subscription updated? I imagine if the user updates the credit card it will trigger this as well

Reply:
Asking here:
https://dailydotdev.slack.com/archives/C07UM5ZRQNL/p1731911775478939

Reply:
Reference doesn't say:
https://developer.paddle.com/webhooks/subscriptions/subscription-updated